src/components/Sidebar.tsx

```diff
@@ -58,9 +59,19 @@ export function Sidebar({
     const pathArray = findPathToContent(fullCourseContent, contentId);
     if (pathArray) {
       const path = `/courses/${courseId}/${pathArray.join('/')}`;
+      console.log(`Path is this ${path}`);
```

Comment:
could you remove this log

Reply:
In the commit named removed debug statements I have removed these logs